spec/models/region_spec.rb

```diff
@@ -0,0 +1,215 @@
+require 'rails_helper'
+
+RSpec.describe Region do
+  describe ".has_any_handbooks?" do
+    let(:region) { FactoryGirl.build(:region) }
+
+    context "for regions with a handbook" do
+      before do
+        region.handbook_url = "http://host.domain"
+      end
+
+      it "returns true" do
+        expect(described_class.has_any_handbooks?([region])).to eq(true)
+      end
+    end
+
+    context "for regions without a handbook" do
+      before do
+        region.handbook_url = nil
+      end
+
+      it "returns false" do
+        expect(described_class.has_any_handbooks?([region])).to eq(false)
+      end
+    end
+  end
+
+  describe "#active_volunteer_count" do
+    subject { FactoryGirl.create(:region) }
+
+    let(:volunteer_1) do
+      FactoryGirl.create(
```

Comment:
It should, because of this line: https://github.com/boulder-food-rescue/food-rescue-robot/blob/master/spec/rails_helper.rb#L25